Packs/DigitalShadows/Classifiers/classifier-Reliaquest_GreyMatter_DRP_Incidents_Mapper.json

```diff
@@ -0,0 +1,222 @@
+					"complex": {
+						"filters": [],
+						"root": "comments",
+						"transformers": [
+							{
+								"operator": "ArrayToCSV"
+							}
+						]
+					}
+				},
+				"DS Alert Description": {
+					"simple": "alert.description"
+				},
+				"DS Alert Email": {
+					"simple": "alert.email"
+				},
+				"DS Alert Event": {
+					"complex": {
+						"filters": [],
+						"root": "comments",
+						"transformers": [
+							{
+								"operator": "ArrayToCSV"
+							}
+						]
+					}
+				},
+				"DS Alert ID": {
+					"simple": "alert.id"
+				},
+				"DS Alert Mitre Attack Mapping": {
+					"complex": {
+						"accessor": "mitre-attack-mapping",
+						"filters": [],
+						"root": "alert",
+						"transformers": [
+							{
+								"operator": "ArrayToCSV"
+							}
+						]
+					}
+				},
+				"DS Alert Password": {
+					"simple": "alert.password"
+				},
+				"DS Alert Risk Factor": {
+					"simple": "alert.risk-factors"
+				},
+				"DS Alert Risk Factors": {
+					"simple": "alert.risk-factors"
+				},
+				"DS Alert Title": {
+					"simple": "alert.title"
+				},
+				"DS Alert Validation": {
+					"complex": {
+						"accessor": "validation",
+						"filters": [],
+						"root": "alert",
+						"transformers": [
+							{
+								"operator": "ArrayToCSV"
+							}
+						]
+					}
+				},
+				"DS Assets": {
+					"complex": {
+						"accessor": "[0]",
+						"filters": [],
+						"root": "assets",
+						"transformers": [
+							{
+								"operator": "ArrayToCSV"
+							}
+						]
+					}
+				},
+				"DS Auto Closed": {
+					"simple": "auto-closed"
+				},
+				"DS Event Action": {
+					"simple": "event.event-action"
+				},
+				"DS Event Number": {
+					"simple": "event.event-num"
+				},
+				"DS Impact Description": {
+					"simple": "incident.impact-description"
+				},
+				"DS Incident Description": {
+					"simple": "incident.description"
+				},
+				"DS Incident ID": {
+					"simple": "incident.id"
+				},
+				"DS Incident Mitigation": {
+					"simple": "incident.impact-description"
+				},
+				"DS Incident Raised Date": {
+					"simple": "incident.raised"
+				},
+				"DS Incident Updated Date": {
+					"simple": "incident.updated"
+				},
+				"DS Mitre Mapping": {
+					"complex": {
+						"accessor": "mitre-attack-mapping",
+						"filters": [],
+						"root": "incident",
+						"transformers": [
+							{
+								"operator": "StringToArray"
+							}
+						]
+					}
+				},
+				"DS Risk Factors ": {
+					"complex": {
+						"accessor": "risk-factors",
+						"filters": [],
+						"root": "incident",
+						"transformers": [
+							{
+								"operator": "ArrayToCSV"
+							},
+							{
+								"args": {
+									"item": {},
+									"raw": {}
+								},
+								"operator": "AppendIfNotEmpty"
+							}
+						]
+					}
+				},
+				"DS Risk Level": {
+					"simple": "triage_item.risk-level"
+				},
+				"DS Risk Type": {
+					"simple": "triage_item.risk-type"
+				},
+				"DS Severity": {
```

Comment:
You've already mapped key into the OOTB field `Severity` - no need for duplication

Reply:
Severity dont have very-low, very-high values so we have created custom fields for that.